string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,53 @@
+/* exported titleCase */
+function titleCase(string) {
```

Comment:
This is a good start on this one, a couple tips on things you can do to help clean it up a bit:

1. Lowercase entire `string` before you do anything else with it, this will give you a consistent starting point.
2. Make some helper functions, there is a lot of repeated code in here, so any task you see being done in multiple places consider making an additional helper function. A good example would be to make a helper function that capitalizes a word. I'm sure there are others...

This is a tough problem and what you have so far if looking good, just see what you can do to clean it up a bit.